### DISCUSSION TITLE
Please add these ENV features

### DISCUSSION THREAD

**Discussion by Capta1nRaj** [vercel/community#7116] [Idea]
1. Export all ENV with 1 click.
2. Show total ENV count.

**Comment by amyegan** [comment]
Hey @Capta1nRaj. You can download a copy of your environment variables using [Vercel CLI](https://vercel.com/docs/cli).

After connecting to the project you'll use [`vercel env pull MY_FILE_NAME`](https://vercel.com/docs/cli/env) to export development environment variables to a local file.
